src/com/palmergames/bukkit/towny/object/Resident.java

```diff
@@ -945,6 +947,14 @@ public void save() {
 		TownyUniverse.getInstance().getDataSource().saveResident(this);
 	}
 
+	public long getJoinedTownAt() {
+		return joinedTownAt;
+	}
```

Comment:
```suggestion
	public long getJoinedTownAt() {
        if (joinedTownAt == null)
            return 0;
		return joinedTownAt;
	}
```

In order to handle players who joined the town prior to this update.